src/main/java/com/alipay/oceanbase/rpc/bolt/transport/ObPacketFactory.java

```diff
@@ -75,6 +75,7 @@ private byte[] encodePayload(ObPayload payload) {
         // us
         rpcHeaderPacket.setTimeout(payload.getTimeout() * 1000);
         rpcHeaderPacket.setTenantId(payload.getTenantId());
+        rpcHeaderPacket.setPrvTenantId(payload.getTenantId());
```

Comment:
What is the difference between tenant ID and prevTenant ID? Why did we set the same tenant id?